reboot_hung_systems_config.json

```diff
@@ -0,0 +1,31 @@
+{
+    "systems": {
+        "otc-gfxtest-bsw-01.local": {"switch": 1, "outlet": 1},
+        "otc-gfxtest-bsw-02.local": {"switch": 1, "outlet": 2},
+        "otc-gfxtest-bsw-03.local": {"switch": 1, "outlet": 3},
+        "otc-gfxtest-bsw-04.local": {"switch": 1, "outlet": 4},
+        "otc-gfxtest-bsw-05.local": {"switch": 1, "outlet": 5},
+        "otc-gfxtest-bsw-06.local": {"switch": 1, "outlet": 6},
+        "otc-gfxtest-bsw-07.local": {"switch": 1, "outlet": 7},
+        "otc-gfxtest-bsw-08.local": {"switch": 1, "outlet": 8},
+        "otc-gfxtest-bsw-09.local": {"switch": 2, "outlet": 7},
+        "otc-gfxtest-glk-01.local": {"switch": 2, "outlet": 1},
+        "otc-gfxtest-glk-02.local": {"switch": 2, "outlet": 2},
+        "otc-gfxtest-glk-03.local": {"switch": 2, "outlet": 3},
+        "otc-gfxtest-glk-04.local": {"switch": 2, "outlet": 4},
+        "otc-gfxtest-glk-05.local": {"switch": 2, "outlet": 5},
+        "otc-gfxtest-glk-06.local": {"switch": 2, "outlet": 6}
+    },
+    "switches": {
+        "1": {
+            "hostname": "192.168.1.2",
+            "userid": "admin",
+            "password": "1234"
+        },
+        "2": {
+            "hostname": "192.168.1.3",
+            "userid": "admin",
+            "password": "1234"
+        }
+    }
+}
```

Comment:
this file should have the config of the perf systems, since this commit is for mesa_perf.